base/src/main/java/io/spine/type/OneofDeclaration.java

```diff
@@ -18,22 +18,18 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.code.gen.java;
+package io.spine.type;
```

Comment:
Why do we change the package?

Reply:
Currently the `type` package contains only few classes that deal with known types.